src/psyclone/tests/domain/gocean/transformations/gocean_opencl_trans_test.py

```diff
@@ -101,15 +92,13 @@ def test_opencl_compiler_works(kernel_outputdir):
       write (*,*) "Hello"
     end program hello
     '''
-    old_pwd = kernel_outputdir.chdir()
-    try:
+
+    with change_dir(kernel_outputdir):
```

Comment:
Use the new fixture instead?

Reply:
I actually thought about this when I changed the code. I decided to use this to be more obvious of what we are changing into. but I believe even if kernel outputdir (which happens to be tmpdir) and cwd=tmpdir is different, it should still work - so I used the fixture here as well.